docs/src/main/paradox/read-journal.md

```diff
@@ -0,0 +1,21 @@
+# Query Plugin
+
+It implements the following [Persistence Queries](https://doc.akka.io/docs/akka/current/scala/persistence-query.html):
+
+* persistenceIds, currentPersistenceIds
+* eventsByPersistenceId, currentEventsByPersistenceId
+* eventsByTag, currentEventsByTag
+
+Persistence Query usage example to obtain a stream with all events tagged with "someTag" with Persistence Query:
+
+    val queries = PersistenceQuery(system).readJournalFor[CassandraReadJournal](CassandraReadJournal.Identifier)
+    queries.eventsByTag("someTag", Offset.noOffset)
```

Comment:
This could be a snippet from https://github.com/akka/akka-persistence-cassandra/blob/master/core/src/multi-jvm/scala/akka/cluster/persistence/cassandra/EventsByTagMultiJvmSpec.scala#L100 instead.

Reply:
We'll need to rework the docs more anyway.